src/validators/LoginRequestValidator.ts

```diff
@@ -2,14 +2,23 @@ import { LoginRequest } from './../models/LoginRequest';
 
 
 export const validateLoginRequest = function (loginReq: LoginRequest): void {
-    if (!validateEmail(loginReq.email)) {
-        throw new Error("Invalid Email Format!");
+
+    if (loginReq.email == '' && loginReq.password == '') {
+        throw new Error("Please enter email and password")
+    }
+
+    if (loginReq.email == '') {
```

Comment:
if not entered is it always empty string or could it be null/undefined?

Reply:
Should probably do a check for null or undefined too. If going through the UI it can only ever be '' if the field is left empty, but if the endpoint gets hit through postman or something it could theoretically get a null or undefined and fall over

Reply:
Chatted on teams, we're happy as is - no requirements for API to be used other than via form.